arangograph/notebooks.md

```diff
@@ -115,8 +119,10 @@ Single line commands have `%` prefix and multi-line commands have `%%` prefix.
 - `%whichCollection` - returns the collection name that is currently selected.
 - `%createCollection collectionName` - creates a collection.
 - `%selectCollection collectionName` - selects a collection as the current collection.
+- `%useCollection collectionName` - 
 - `%getCollection collectionName` - gets a collection for variable assignment,
   e.g. `student` = `% getCollection Student`.
+- `%createEdge` - creates an edge collection.    
```

Comment:
If this command actually creates an edge collection and not an edge, then its name is confusing.

Reply:
@cw00dw0rd can you please clarify?

Reply:
It adds an edge definition to a graph. This was added by Arthur, I will verify the desired use case and follow-up, maybe with a better name or different functionality.

Reply:
Update on this:
- Name will be adjusted to `createEdgeDefintion`
- Functions for `createEdgeCollection` and `createVertexCollection` will be added to further distinguish functionality

Reply:
I will come back here once I have pushed the new image, likely Thursday, to verify they have been added

Reply:
Yes, please let us know once they are added. thanks 👍